deps/build.jl

```diff
@@ -0,0 +1,4 @@
+default_dir = get(ENV, "CONDA_JL_HOME", abspath(joinpath(dirname(@__FILE__), "usr")))
```

Comment:
Because when you re-run Pkg.build, you don't read deps.jl

Imagine where the user does `ENV["CONDA_JL_HOME"]="....."; Pkg.build("Conda")` and is happy with her Conda configuration.  Then, at some point later, in a separate Julia session, she does `Pkg.update()`, which updates Conda and internally calls `Pkg.build("Conda")`.  Since the `CONDA_JL_HOME` is no longer set, it forgets her setting and reverts to the default configuration.

Instead, the build script needs to remember `default_dir` from previous builds, and use it unless the `CONDA_JL_HOME` environment variable is explicitly set.